<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>339</width>
    <height>357</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QCompleter Example</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox2">
      <property name="title">
       <string>Input Fields</string>
      </property>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label1">
         <property name="text">
          <string>Line edit:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEdit"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label2">
         <property name="text">
          <string>Combo box:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="comboBox"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox1">
      <property name="title">
       <string>Settings</string>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label3">
         <property name="text">
          <string>Completion mode:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="completionModeComboBox">
         <item>
          <property name="text">
           <string>Popup</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Inline</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Unfiltered Popup</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label4">
         <property name="text">
          <string>Model sorting:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="modelSortingComboBox">
         <item>
          <property name="text">
           <string>Unsorted</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Case Sensitively Sorted</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Case Insensitively Sorted</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label6">
         <property name="text">
          <string>Filter mode:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="filterModeComboBox">
         <item>
          <property name="text">
           <string>Starts With</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Contains</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Ends With</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label5">
         <property name="text">
          <string>Max visible items:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QSpinBox" name="maxVisibleItemsSpinBox">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="value">
          <number>7</number>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QCheckBox" name="caseSensitiveCheckBox">
         <property name="text">
          <string>Case sensitive</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>339</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
